src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmEvaluationAsyncResult.cs

```diff
@@ -27,9 +27,9 @@ public DkmEvaluationAsyncResult(DkmEvaluationResult Result)
             _result = Result;
         }
 
-        public int ErrorCode { get { throw new NotImplementedException(); } }
+        public readonly int ErrorCode { get { throw new NotImplementedException(); } }
```

Comment:
💡 We should consider _not_ reporting the diagnostic for members that just throw `NotImplementedException` for two reasons:

1. These members, when implemented, seem reasonably likely to no longer qualify for the diagnostic
2. There is no value in optimizing for a call to a method that unconditionally throws
3. This will avoid a wall of diagnostics that immediately appears after every use of the Implement Interface refactoring

Reply:
will change this.  thanks.